tests/phpunit/tests/post/revisions.php

```diff
@@ -655,6 +655,55 @@ public function test_wp_save_post_revision_error() {
 		$this->assertWPError( $revision );
 	}
 
+	/**
+	 * Tests that wp_get_last_revision_id_and_total_count() returns last revision id and total count.
+	 *
+	 * @ticket 55857
+	 * @dataProvider data_wp_get_post_revisions_url
+	 */
+	public function test_wp_get_last_revision_id_and_total_count( $revisions ) {
+		wp_set_current_user( self::$admin_user_id );
+		$post_id            = self::factory()->post->create( array( 'post_title' => 'Some Post' ) );
```

Comment:
```suggestion
		$post_id            = self::factory()->post->create();
```
Do we really need this title?